migrations.lock

```diff
@@ -1201,6 +1201,7 @@ users
  0055_add_user_data
  0056_add_manage_domain_alerts_permission
  0057_populate_sql_user_data
+ 0058_auto_20240327_0011
```

Comment:
Nitpick: a human readable name would be helpful.

Reply:
[233cf8a](https://github.com/dimagi/commcare-hq/pull/34341/commits/233cf8a9e2816d22bab3a2ea26c24fd4770cd751)